### DISCUSSION TITLE
How do we best share that new zim files are available?

### DISCUSSION THREAD

**Discussion by Popolechien** [openzim/zim-requests#322] [Ideas]
Hola,

the first item for discussion would be how to best broadcast which new zim files are out there.

Currently the process is:
- someone posts a request on /zim-requests
- the request is turned into a recipe on the [zimfarm](https://farm.openzim.org)
- if it worked the request is closed and the requester gets an automated notification from Github

Now there's potentially plenty of other people that might be interested by this new content but do not know about it. How do we tell them, and how do we manage to make sure it is systematic / regularly done with our limited resources? that last point is important because yes, posting on Twitter, Facebook, Reddit, and sending out a newsletter every week would be best, but that takes time which adds up for every step and that we may not have.

I'm welcoming ideas of what would be most impactful in terms of reach, and what tools would be most helpful (e.g. is there a free tool of sorts that would allow us to post the same thing on multiple channels), and what a proper process could be.

Feel free to brainstorm and shoot random ideas, at this stage there is no stupid and we have no clear plan. Just a problem in need of a solution.

**Comment by dattaz** [comment]
I don't know too much about, but maybe a github bot can do the job ? Something that when you close an issue (maybe by specific people, or with specific tag and certainly with a comment of a certain format) automatically send twitter/reddit/slack/irc/rss/log somewhere. And then with the log somewhere (maybe a simple php page (or something else more appropriate) that handles this) trigger the automatic sending of a mail every X time.

I think we already have an OPDS with everything publish ? Maybe we can use it also (Find new "name" inside the xml each X time)

**Reply by Popolechien** [reply]
A similar idea would be to check if IFTTT does have something like it.

**Reply by Popolechien** [reply]
IFTTT _does_ have such function but it's a very simple one and would not distinguish whether a ticket is being closed successfully or not :-(

**Comment by kelson42** [comment]
This question implies 3 sub questions (and two of them are clearly answered I believe):
* What is the new content? This is decided based on the ZIM "name" metadata. This is the case when we never have published a ZIM with such a name.
* Which system is in charge of telling a new content is there? The same which decides that new ZIM are published: the (to be built) CMS. So for sure not the Zimfarm (no publishing role).
* Last question is the most interesting, how people want to be notified (and how to configure it)?
